src/mobile/apps/artwork/components/partner/query.coffee

```diff
@@ -1,22 +1,24 @@
 module.exports = """
   partner {
-    id
-    name
-    href
-    type
+    acceptsCardPayments
```

Comment:
@ashkan18 - this is what get added for all artworks that eventually makes a call to lewitt through MP.